### ISSUE TITLE
"ajaxExtended call failed" error during Azure Bot creation

### ISSUE THREAD

**Issue by SUDALV92** [Microsoft/BotBuilder#4444]
## Bot Info
* Deployment Environment:  Azure Bot Service 

## Issue Description
I got "ajaxExtended call failed" error when i'm trying to create Azure Web App bot. Sometimes bot creation going without any errors. It does not depend on the parameters.

## Reproduction Steps
1. Start creating Web App Bot in Azure Portal
2. Set the parameters (in my case I'm using an existing resource group, creating Asure Storage)
3. Press "create" button

## Expected Behavior
The bot should be created without any errors.

## Actual Results
![ss 2018-04-09 at 04 01 09](https://user-images.githubusercontent.com/38214332/38499117-42ed0a7c-3c0f-11e8-90b5-a3328362fc57.png)

**Comment by EricDahlvang** [comment]
When this happens, do you see any errors in the console window of the browser (F12)?

**Comment by SUDALV92** [comment]
Yes
![ss 2018-04-10 at 12 59 38](https://user-images.githubusercontent.com/38214332/38550454-46c4740e-3cbf-11e8-9caa-c1e694037f3b.png)

**Comment by benjamin-p-dennis** [comment]
I'm experiencing this issue also. Enterprise support confirmed this is a known issue.

**Comment by EricDahlvang** [comment]
Closing as a fix for this has been deployed.

**Comment by georearl** [comment]
This is still occurring to me today

**Comment by mrRobot62** [comment]
same on my side :-( - can't create a bot

`
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (invoke, line 0)
[Error] [Microsoft_Azure_BotService]  – "18:46:30" – "MsPortalFx.Base.Diagnostics.ErrorReporter" – 1 – "MsPortalFx.Base.Diagnostics.ErrorReporter: _errorData: undefined
↵_sourceErrorLevel: undefined
↵baseTypes…"
	r (2dLd3nZ3cXG9.js:6:2452)
	v (2dLd3nZ3cXG9.js:6:1852)
	p (2dLd3nZ3cXG9.js:6:3489)
	error (2dLd3nZ3cXG9.js:6:4368)
	g (2dLd3nZ3cXG9.js:32:113)
	b (2dLd3nZ3cXG9.js:32:1249)
	(anonyme Funktion) (2dLd3nZ3cXG9.js:32:856)
	forEach
	p (2dLd3nZ3cXG9.js:32:818)
[Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error) (createLuisApp, line 0)
[Error] [Microsoft_Azure_BotService]  – "18:46:41" – "MsPortalFx/Base/Base.Net" – 1 – "Base.Net: readyState: 4
↵responseJSON: {\"Error\":{\"Code\":\"ServiceError\",\"Message\":null,\"InnerHttpError\":null}}
↵responseText…" – {uri: undefined, type: "POST", pathAndQuery: undefined, …}
{uri: undefined, type: "POST", pathAndQuery: undefined, requestId: "634a5b30-51e2-4c10-be7f-4849eb98a062", failureCause: undefined, …}Object
	r (2dLd3nZ3cXG9.js:6:2452)
	v (2dLd3nZ3cXG9.js:6:1852)
	p (2dLd3nZ3cXG9.js:6:3489)
	error (2dLd3nZ3cXG9.js:6:4368)
	(anonyme Funktion) (2dLd3nZ3cXG9.js:42:8527)
	(anonyme Funktion) (BUbKUXDk0t3l.js:45:40335)
	c (BUbKUXDk0t3l.js:45:39009)
	fireWith (BUbKUXDk0t3l.js:45:39773)
	b (BUbKUXDk0t3l.js:45:74892)
	(anonyme Funktion) (BUbKUXDk0t3l.js:45:80722)
[Error] [Microsoft_Azure_BotService]  – "18:46:41" – "Resource/Create/ViewModels/CreateBladeViewModel" – 5 – "CreateBladeViewModel: Failed creating LUIS app" – "_errorData: undefined
↵_sourceErrorLevel: undefined
↵baseTypes: [\"MsPortalFx.Errors.AjaxError\",\"MsPortalFx.Errors.Error\"]
↵code: undefined…"
	r (2dLd3nZ3cXG9.js:6:2452)
	v (2dLd3nZ3cXG9.js:6:1852)
	p (2dLd3nZ3cXG9.js:6:3489)
	error (2dLd3nZ3cXG9.js:6:4368)
	(anonyme Funktion) (bV0GrBomRTyO.js:10:1075)
	c (BUbKUXDk0t3l.js:30:8631)
	(anonyme Funktion) (BUbKUXDk0t3l.js:30:8873)
	promiseDispatch (BUbKUXDk0t3l.js:30:2908)
	(anonyme Funktion) (BUbKUXDk0t3l.js:30:1686)
	t (BUbKUXDk0t3l.js:30:6269)
`

**Comment by dranreb** [comment]
still not working.

POST https://botservice.hosting.portal.azure.net/api/ClientTrace?appCacheKind=None&bundlingKind=PopularPartitioner&cacheability=3&extensionName=Microsoft_Azure_BotService&l=en.en-us&pageVersion=1.1.0.7&trustedAuthority=portal.azure.com 404 ()
send @ BUbKUXDk0t3l.js:45
ajax @ BUbKUXDk0t3l.js:45
t.ajax @ 2dLd3nZ3cXG9.js:17
u._worker @ 2dLd3nZ3cXG9.js:42
h @ 2dLd3nZ3cXG9.js:42
(anonymous) @ 2dLd3nZ3cXG9.js:42
h @ BUbKUXDk0t3l.js:30
(anonymous) @ BUbKUXDk0t3l.js:30
f.promiseDispatch @ BUbKUXDk0t3l.js:30
(anonymous) @ BUbKUXDk0t3l.js:30
t @ BUbKUXDk0t3l.js:30
2dLd3nZ3cXG9.js:6 [Microsoft_Azure_BotService]  9:59:07 AM MsPortalFx.Base.Diagnostics.ErrorReporter 1 MsPortalFx.Base.Diagnostics.ErrorReporter: _errorData: undefined
_sourceErrorLevel: undefined
baseTypes: ["MsPortalFx.Errors.AjaxError","MsPortalFx.Errors.Error"]
code: undefined
data: {"type":"POST","requestId":"b45c56b6-8d40-4afd-95d3-e9342b331025","sessionId":"c4a50aef3bbf410385985615ea48ea4e","status":404,"statusText":"error","duration":110.10000000533182}
errorLevel: 2
errorThrown: 
extension: Microsoft_Azure_BotService
handled: undefined
innerErrors: []
jqXHR: {"readyState":4,"responseText":"Cannot POST /api/ClientTrace?appCacheKind=None&amp;bundlingKind=PopularPartitioner&amp;cacheability=3&amp;extensionName=Microsoft_Azure_BotService&amp;l=en.en-us&amp;pageVersion=1.1.0.7&amp;trustedAuthority=portal.azure.com\n","status":404,"statusText":"error"}
message: ajaxExtended call failed
name: Error
source: undefined
stack: null
textStatus: error
timestamp: 166829.20000000013
type: MsPortalFx.Errors.AjaxError

**Comment by EricDahlvang** [comment]
@mrRobot62  and  @dranreb Thank you for reporting this.  We are investigating.  

Which bot template are you both using?

**Comment by dranreb** [comment]
I was using LUISOn Jul 11, 2018 3:09 PM, Eric Dahlvang <notifications@github.com> wrote:@mrRobot62  and  @dranreb Thank you for reporting this.  We are investigating.
Which bot template are you both using?

**Comment by EricDahlvang** [comment]
The Azure Bot Service engineering team confirmed that there was an intermittent disruption in the service causing bot creation failures. At this time, the issue has been mitigated so please retry your create request. If you still having problems, please do let us know by opening a new issue: https://github.com/Microsoft/BotBuilder/issues/new

**Comment by SiddharthSinghMOD** [comment]
I still see the same error today while creating a LUIS bot.
Any solution.. how can I fix this.
I tried several names for bot.. none worked.

**Comment by EricDahlvang** [comment]
@SiddharthSinghMOD  Please see  #4893 and #4892

**Comment by hinojosachapel** [comment]
This issue is still happening. I'm unable to create a new Web App Bot (LUIS template) :-(

**Before starting the process**
![01 before starting process](https://user-images.githubusercontent.com/17761991/44085451-e720efba-9fb9-11e8-95fa-baa5264d9d0b.png)

**After starting the process**
![02 after starting process](https://user-images.githubusercontent.com/17761991/44085459-ed9a8fd6-9fb9-11e8-8853-da5bb0b957f5.png)

**Comment by hinojosachapel** [comment]
Maybe the current issue relates with this one, happening concurrently (luis.ia). None of the members of my team is able to sign in:

![luis ai](https://user-images.githubusercontent.com/17761991/44086464-ee2b7a66-9fbc-11e8-8675-091594ba0909.png)

**Comment by k00bas** [comment]
Same error happened on my end, validation took 2 minutes approx., afterwards an error message:
![image](https://user-images.githubusercontent.com/17956043/44955018-88f27880-aeac-11e8-8351-347b381e3deb.png)
According to Azure Status page all the necessary services in a region needed were in "good" state

**Comment by s8surajds** [comment]
Change the Name and try again.
It works when this is done

**Comment by ishancoderr** [comment]
![error](https://user-images.githubusercontent.com/31079429/149675684-ed9d5598-5aa7-426d-abcf-cca652effb15.PNG)
